medical_pharmacy/README.rst

```diff
@@ -1,45 +1,47 @@
 .. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 
+=====================
 Odoo Medical Pharmacy
-========================
-
-Extension of medical that adds a pharmacy namespace on partners
+=====================
 
+Extension of medical that adds a pharmacy namespace on partners.
 
 Usage
 =====
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/159/8.0
-
-For further information, please visit:
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
 
-* https://www.odoo.com/forum/help-1
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```

Comment:
No, just the part I noted. Those are instructions to creating the Runbot URI, which is already done in your template.